components/admin/lessons/AdminModuleInputs.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { useEffect, useState } from 'react'
+import {
+  AddModuleMutation,
+  UpdateModuleMutation,
+  useAddModuleMutation,
+  useUpdateModuleMutation
+} from '../../../graphql'
+import { formChange } from '../../../helpers/formChange'
+import { FormCard, MD_INPUT, TextField } from '../../FormCard'
+import { AlertFillIcon, CheckCircleIcon } from '@primer/octicons-react'
+import styles from '../../../scss/adminModuleInputs.module.scss'
+import { Spinner } from 'react-bootstrap'
+import { get } from 'lodash'
+import { RefetchQueriesFunction, ApolloError } from '@apollo/client'
+
+type Module = { id: number; name: string; content: string }
+
+type Props = {
+  lessonId: number
+  refetch: RefetchQueriesFunction
```

Comment:
This is a `useQuery` refetch function. It's used to refetch the modules and update the modules list with the newly added modules.